Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -146,6 +146,10 @@ class BrowserViewController: UIViewController {
     private(set) var publisher: PublisherInfo?
     
     let vpnProductInfo = VPNProductInfo()
+    
+    /// Tracking If a product notification is presented in order to not to try to present another one over existing popover
+    var benchmarkNotificationPresented: Bool
```

Comment:
if this is set to `false` directly in `init`, why not just set it `= false` here at declaration?